docs/conf.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
```

Comment:
You can drop these coding cookies as well. Python 3 defaults to decoding source files as utf-8.